src/integrations/Clevertap/browser.js

```diff
@@ -60,7 +62,12 @@ class Clevertap {
       onUserLogin: [],
       notifications: [],
     };
+    const { CLEVERTAP } = this.analytics.loadOnlyIntegrations;
+    this.optOut = CLEVERTAP.optOut || false;
+    this.useIP = CLEVERTAP.useIP || false;
     window.clevertap.enablePersonalization = true;
+    window.clevertap.privacy.push({ optOut: this.optOut }); // set the flag to true, if the user of the device opts out of sharing their data
+    window.clevertap.privacy.push({ useIP: this.useIP }); // set the flag to true, if the user agrees to share their IP data
```

Comment:
```suggestion
    window.clevertap.privacy.push({ optOut: this.optOut }); 
    window.clevertap.privacy.push({ useIP: this.useIP });
```
These comments are not required rather you can share the SDK doc of clevertap.

Reply:
updated